tests/test_data_parallel.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+import numpy as np
+
+from devito import Grid, Function
+from devito.data import loc_data_idx
+
+
+class TestDataParallel(object):
```

Comment:
Can maybe be moved in `test_mpi`?

Reply:
I've moved them to `TestDistributedData` in `test_data.py` now.